### PR TITLE
Improve tools.json doc to emphasize SemVer vs. chronological sorting

### PR DESCRIPTION
**Pull request by joelverhagen**

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2254f2c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2254f2c5280dea8283291d9285a14bed966fc1be)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2254f2c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2254f2c5280dea8283291d9285a14bed966fc1be)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2254f2c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2254f2c5280dea8283291d9285a14bed966fc1be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/tools-json.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-toolsjson/docs/api/tools-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/tools-json?branch=pr-en-us-1315) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902141836206186-1315%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SSrfx%2F8SCNWxIa43Iotx9HfBgwu7H8H%2F3hwzZBWNoVc%3D&st=2019-02-14T18%3A32%3A28Z&se=2019-03-17T18%3A37%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/tools-json.md [hunk 1]
```diff
@@ -54,10 +54,16 @@ url      | string | yes      | An absolute URL for downloading this version of n
 stage    | string | yes      | An enum string
 uploaded | string | yes      | An approximate timestamp of when the version was made available
 
-The items in the array will be sorted in descending, SemVer 2.0.0 order. This guarantee is meant to ease the burden on
-a client looking for the latest version. 
+The items in the array will be sorted in descending, SemVer 2.0.0 order. This guarantee is meant to reduce the burden
+of a client that is interested in highest version number. However this does mean that the list is not sorted in
+chronological order. For example, if an lower major version is serviced at a date later than a higher major version,
```
Comment (loic-sharma) [inline docs/api/tools-json.md]: `an lower` -> `a lower`

File: docs/api/tools-json.md [hunk 2]
```diff
@@ -54,10 +54,16 @@ url      | string | yes      | An absolute URL for downloading this version of n
 stage    | string | yes      | An enum string
 uploaded | string | yes      | An approximate timestamp of when the version was made available
```
Comment (loic-sharma) [inline docs/api/tools-json.md]: In the uploaded notes, can we mention this is ISO 8601 format?

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2767a56](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2767a56dc3ff988fc59d7339104da289a6298c42)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2767a56](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2767a56dc3ff988fc59d7339104da289a6298c42)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[2767a56](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1315/commits/2767a56dc3ff988fc59d7339104da289a6298c42)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/tools-json.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-toolsjson/docs/api/tools-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/tools-json?branch=pr-en-us-1315) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902141933377244-1315%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fgvn3%2BZc0Jop%2BtpteVzbO2RQhTH04StsHUfTn4qoxis%3D&st=2019-02-14T19%3A29%3A40Z&se=2019-03-17T19%3A34%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**